INSTALL.md

```diff
@@ -198,6 +198,44 @@ Then to compile and install do this:
 sudo ./scripts/install-dependencies
 ```
 
+## Apparmor Profile (Ubuntu 24.04+)
+
+Beginning with the 24.04 LTS release, Ubuntu does not permit applications to
+create unprivileged user namespaces by default.
+
+If you install Apptainer from a GitHub release `.deb` package then an
+apparmor profile will be installed that permits Apptainer to create
+unprivileged user namespaces.
+
+If you install Apptainer from source you must configure apparmor.
```

Comment:
Change `you must configure apparmor` to `you must either configure apparmor or disable the restriction on unprivileged user namespaces`.  Then start a new paragraph and change the next sentence to `To create an apparmor profile:`.  Then add this to the end of the section:

~~~
If instead you want to allow unprivileged users to create any unprivileged user namespace do the following:

```sh
sudo sh -c 'echo kernel.apparmor_restrict_unprivileged_userns=0 \
    >/etc/sysctl.d/90-disable-userns-restrictions.conf'
sudo sysctl -p /etc/sysctl.d/90-disable-userns-restrictions.conf
``` 
~~~